Gemfile

```diff
@@ -110,7 +110,7 @@ local_gemfile = File.expand_path(".Gemfile", __dir__)
 instance_eval File.read local_gemfile if File.exist? local_gemfile
 
 group :test do
-  gem "minitest", "= 5.11.1"
+  gem "minitest"
```

Comment:
Can we remove this dependency entirely now that it needn't be pinned to a specific version?

Reply:
Yup, good point